proto/interchain_security/ccv/provider/v1/query.proto

```diff
@@ -372,10 +375,11 @@ message QueryConsumerChainRequest {
 }
 
 message QueryConsumerChainResponse {
-  string chain_id = 1;
-  string owner_address = 2;
-  string phase = 3;
-  ConsumerMetadata metadata = 4 [ (gogoproto.nullable) = false ];
-  ConsumerInitializationParameters init_params = 5;
-  PowerShapingParameters power_shaping_params = 6;  
+  string consumer_id = 1;
```

Comment:
So that we can validate on the FE that we got the chain_id for the consumer_id we requested. But, more importantly so that we don't have to keep extra context (e.g coming from URL params) about which chain we're dealing with.

E.g. previously we had to read the URL param for consumer_id for any operation we were doing:
```
forge/consumer-id/123
```

So we had to "remember" the `123`. This way we look up an object and treat is as the source of truth and do all operations required by fetching the consumer_id from the returned object.

Does that make sense?